src/lang/builtins_json.ml

```diff
@@ -372,10 +305,9 @@ let () =
       Lang.meth (JsonValue.to_value v) meth)
 
 let () =
-  Lang.add_builtin "_internal_json_renderer_" ~category:`String ~flags:[`Hidden]
-    ~descr:"Internal JSON renderer."
+  Lang.add_builtin "json.stringify" ~category:`String
+    ~descr:"Convert a value to JSON when possible."
```

Comment:
The "when possible" is a bit scary: explain " (values like functions cannot be converted to JSON)".